From e63631c58b14ce2d1b481b88b511164ec3fb568d Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Fri, 20 Jun 2014 18:53:01 +0200
Subject: ARM: shmobile: Move r7s72100.h

Change location of r7s72100.h so it can be included using "r7s72100.h"
instead of the old style <mach/r7s72100.h>.

Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit 96bd090066e2150f3970b32f35895de2ab631c95)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 arch/arm/mach-shmobile/board-genmai-reference.c      | 4 +++-
 arch/arm/mach-shmobile/board-genmai.c                | 4 +++-
 arch/arm/mach-shmobile/clock-r7s72100.c              | 3 ++-
 arch/arm/mach-shmobile/{include/mach => }/r7s72100.h | 0
 arch/arm/mach-shmobile/setup-r7s72100.c              | 4 +++-
 5 files changed, 11 insertions(+), 4 deletions(-)
 rename arch/arm/mach-shmobile/{include/mach => }/r7s72100.h (100%)

diff --git a/arch/arm/mach-shmobile/board-genmai-reference.c b/arch/arm/mach-shmobile/board-genmai-reference.c
index c06dc5c12d1e..e5448f7b868a 100644
--- a/arch/arm/mach-shmobile/board-genmai-reference.c
+++ b/arch/arm/mach-shmobile/board-genmai-reference.c
@@ -20,11 +20,13 @@
 
 #include <linux/kernel.h>
 #include <linux/of_platform.h>
-#include <mach/r7s72100.h>
+
 #include <asm/mach-types.h>
 #include <asm/mach/arch.h>
+
 #include "clock.h"
 #include "common.h"
+#include "r7s72100.h"
 
 /*
  * This is a really crude hack to provide clkdev support to platform
diff --git a/arch/arm/mach-shmobile/board-genmai.c b/arch/arm/mach-shmobile/board-genmai.c
index a0994f36b348..e2a3ba4871c3 100644
--- a/arch/arm/mach-shmobile/board-genmai.c
+++ b/arch/arm/mach-shmobile/board-genmai.c
@@ -25,11 +25,13 @@
 #include <linux/sh_eth.h>
 #include <linux/spi/rspi.h>
 #include <linux/spi/spi.h>
-#include <mach/r7s72100.h>
+
 #include <asm/mach-types.h>
 #include <asm/mach/arch.h>
+
 #include "common.h"
 #include "irqs.h"
+#include "r7s72100.h"
 
 /* Ether */
 static const struct sh_eth_plat_data ether_pdata __initconst = {
diff --git a/arch/arm/mach-shmobile/clock-r7s72100.c b/arch/arm/mach-shmobile/clock-r7s72100.c
index 457dab0f8fc9..3eb2ec401e0c 100644
--- a/arch/arm/mach-shmobile/clock-r7s72100.c
+++ b/arch/arm/mach-shmobile/clock-r7s72100.c
@@ -19,8 +19,9 @@
 #include <linux/io.h>
 #include <linux/sh_clk.h>
 #include <linux/clkdev.h>
-#include <mach/r7s72100.h>
+
 #include "common.h"
+#include "r7s72100.h"
 
 /* Frequency Control Registers */
 #define FRQCR		0xfcfe0010
diff --git a/arch/arm/mach-shmobile/include/mach/r7s72100.h b/arch/arm/mach-shmobile/r7s72100.h
similarity index 100%
rename from arch/arm/mach-shmobile/include/mach/r7s72100.h
rename to arch/arm/mach-shmobile/r7s72100.h
diff --git a/arch/arm/mach-shmobile/setup-r7s72100.c b/arch/arm/mach-shmobile/setup-r7s72100.c
index a55d68d89e6e..4a98b232d316 100644
--- a/arch/arm/mach-shmobile/setup-r7s72100.c
+++ b/arch/arm/mach-shmobile/setup-r7s72100.c
@@ -22,10 +22,12 @@
 #include <linux/kernel.h>
 #include <linux/of_platform.h>
 #include <linux/sh_timer.h>
-#include <mach/r7s72100.h>
+
 #include <asm/mach/arch.h>
+
 #include "common.h"
 #include "irqs.h"
+#include "r7s72100.h"
 
 static struct resource mtu2_resources[] __initdata = {
 	DEFINE_RES_MEM(0xfcff0000, 0x400),
-- 
2.1.2

